cockpit-podman.spec.in

```diff
@@ -22,7 +22,7 @@ Summary:        Cockpit component for Podman containers
 License:        LGPLv2+
 URL:            https://github.com/cockpit-project/cockpit-podman
 
-Source0:        https://github.com/cockpit-project/cockpit-podman/releases/download/%{version}/cockpit-podman-%{version}.tar.xz
+Source0: https://github.com/cockpit-project/%{name}/releases/download/%{version}/%{name}-%{version}.tar.xz
```

Comment:
Missing spacing?

Reply:
Whoops, yes! vim pilot error. (not technically important, but unsightly). Fixed